addons/website/tests/test_ui.py

```diff
@@ -62,6 +62,39 @@ def test_01_attachment_website_unlink(self):
 
 @odoo.tests.tagged('-at_install', 'post_install')
 class TestUiHtmlEditor(odoo.tests.HttpCase):
+
+    def test_html_editor_language(self):
+        Page = self.env['website.page']
+
+        default_website = self.env.ref('website.default_website')
+        nl_lang = self.env['res.lang']._activate_lang('nl_NL')
```

Comment:
This should ideally have been `ParselTongue` to not slow down the test as you don't really need the `.pot` files to be loaded.

Could you change that in another PR?

Reply:
@rdeodoo Done in https://github.com/odoo/odoo/pull/144948.